gitio.zsh

```diff
@@ -14,21 +14,35 @@ function gitio() {
 
   # Show help message with colors
   function gitio-help() {
-    printf "USAGE: gitio <[%s]%s/%s> [%s] \n" \
+    printf "USAGE: gitio [[%s]%s/%s] [%s] \n" \
       "https://github.com/" \
       "${fg[cyan]}username${reset_color}" \
       "${fg[magenta]}repo${reset_color}" \
       "${fg[green]}vanity code${reset_color}" \
   }
 
+  # Get URL from git config
+  if [[ "$#" -eq 0 && -s ".git/config" ]]; then
+    local CONFIG_URL="$(
```

Comment:
This is amazing idea! But there is a problem with ssh:

![image](https://cloud.githubusercontent.com/assets/3459374/13599776/923d5ce8-e52d-11e5-9fa5-eabec2085411.png)